vnode_controller/vnode_controller.go

```
@@ -183,25 +188,29 @@
-				}
 			})
 
 			// Periodically check for nodes that are not reachable and notify their leader virtual nodes.
-			go utils.TimedTaskWithInterval(ctx, time.Second, func(ctx context.Context) {
-				unReachableVNode := vNodeController.vNodeStore.GetUnReachableVNodes()
-				if unReachableVNode != nil && len(unReachableVNode) > 0 {
-					nodeNames := make([]string, 0, len(unReachableVNode))
-					for _, vNode := range unReachableVNode {
+			go utils.TimedTaskWithInterval(ctx, 3*time.Second, func(ctx context.Context) {
+				unReachableVNodes := vNodeController.vNodeStore.GetUnReachableVNodes()
+				if unReachableVNodes != nil && len(unReachableVNodes) > 0 {
```

golangci-lint (1.61.0)

S1009: should omit nil check; len() for []*github.com/koupleless/virtual-kubelet/provider.VNode is defined as zero

(gosimple)